config/routes.ts

```diff
@@ -0,0 +1,43 @@
+import buildings from "../src/public/data/buildings.json";
+import spaces from "../src/public/data/spaces.json";
+const buildingMap = {} as Record<number, any>;
+for (const building of buildings) buildingMap[building.number] = building;
+
+//TODO: rewrite after getting rid of vue-i18n
+const routes = [
+  "/en/",
+  "/en/buildings/",
+  "/en/help/",
+  "/en/feedback/",
+  "/nl/",
+  "/nl/gebouwen/",
+  "/nl/hulp/",
+  "/nl/feedback/",
+];
+
+for (const building of buildings) {
+  routes.push("/en/buildings/" + building.i18n.en.slug + "/");
```

Comment:
Use string interpolation instead of concatenation everywhere in this file, since we're using that everywhere else in the project now.